src/langcheck/metrics/model_manager/_model_management.py

```diff
@@ -144,22 +145,25 @@ def validate_config(config, language='all', metric='all') -> None:
                     raise ValueError(
                         f'loader type should in {VALID_LOADER_FUNCTION}')
 
-                # Check model availability with revision if specified.
-                model_name = model_setting.get('model_name')
-                model_revision = model_setting.get('model_revision')
-                if not check_model_availability(model_name, model_revision):
-                    raise ValueError(
-                        f'Cannot find {model_name} with {model_revision} at Huggingface Hub'  # NOQA:E501
-                    )
-                # Check tokenizer availability with revision if specified.
-                tokenizer_name = model_setting.get('tokenizer_name')
-                if tokenizer_name is not None and tokenizer_name != model_name:
-                    tokenizer_revision = model_setting.get('tokenizer_revision')
-                    if not check_model_availability(tokenizer_name,
-                                                    tokenizer_revision):
-                        raise ValueError(
-                            f'Cannot find {tokenizer_name} with {tokenizer_revision} ay Huggingface Hub'  # NOQA:E501
-                        )
+                # TODO: Check the model and tokenizer availability
```

Comment:
Instead of commenting it out, maybe we can add a param (e.g. `run_check_model_availability`) to `validate_config`, set it to False by default, and add an `if run_check_model_availability` statement here? That way we won't run it during tests, but users can check availability if they want to

Reply:
Sounds good, fixed the code in that way